src/test/java/DemoTest.java

```diff
@@ -1,7 +1,125 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
 public class DemoTest {
 
+  @Test
+  public void incorrectPassAndEmailTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://deens-master.now.sh");
+
+    driver.findElement(By.cssSelector("a[href*='login']")).click();
+
+    WebElement email = driver.findElement(By.cssSelector("#email"));
+    email.sendKeys("valery@gmail.com");
+    WebElement pass = driver.findElement(By.cssSelector("#password"));
+    pass.sendKeys("Testing123");
+    driver.findElement(By.cssSelector("[data-testid='loginSubmit']")).click();
+    WebElement errorMessage = driver.findElement(By.cssSelector(".ui.error.message p"));
+    String errorMessageText = errorMessage.getText();
+
+    Assert.assertEquals(errorMessageText, "Wrong email or password.");
+    driver.quit();
+  }
+
+  @Test
+  public void loginEmptyParametrsTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.get("https://deens-master.now.sh/login");
+
+    WebElement loginButton = driver.findElement(By.cssSelector("[data-testid='loginSubmit']"));
```

Comment:
it is not necessary to init login button because you use it only once. You can just write it in one line:
driver.findElement(By.cssSelector("[data-testid='loginSubmit']")).click();

Reply:
done